src/gax/Cargo.toml

```diff
@@ -44,11 +44,12 @@ echo-server = { path = "echo-server" }
 # This is a workaround to integration test features of this crate. Open issue
 # https://github.com/rust-lang/cargo/issues/2911.
 gax         = { path = ".", package = "gcp-sdk-gax", features = ["unstable-sdk-client", "unstable-stream"] }
+async-trait = "0.1.83"
```

Comment:
Do we need this?  It seems like it is not used.

Reply:
Nope, removed.